imports.go

```diff
@@ -20,7 +20,9 @@ import (
 	_ "github.com/mholt/caddy-l4/modules/l4echo"
 	_ "github.com/mholt/caddy-l4/modules/l4http"
 	_ "github.com/mholt/caddy-l4/modules/l4proxy"
+	_ "github.com/mholt/caddy-l4/modules/l4proxyprotocol"
```

Comment:
Oh, was this just not working cause this was missing? Huh.

Reply:
Yeah, definitely! I just have no idea why it was OK when I tested it in #43 🤔

Reply:
Well, oops 😅

Reply:
Oh, I got it! Because it has been imported by l4proxy:

https://github.com/mholt/caddy-l4/blob/d1f4337132468dfe2d081baba3769dfd1f46fbf8/modules/l4proxy/proxy.go#L31